src/module/apps/trade-dialog/app.svelte

```diff
@@ -0,0 +1,448 @@
+<script lang="ts">
+    import { ErrorPF2e } from "@util";
+    import type { TradeQueryData, TradeDialogRenderContext, TradeItemData } from "./app.ts";
+
+    const { state: data, foundryApp: dialog, traderUser, searchEngine, localize }: TradeDialogRenderContext = $props();
+    const { self, trader } = data;
+    const portraitAlt = {
+        self: localize("ImgAlt.Actor", { actor: self.actor.name, user: localize("You") }),
+        trader: localize("ImgAlt.Actor", { actor: trader.actor.name, user: traderUser.name }),
+    };
+
+    // Item lists and summaries
+    const listFormatter = game.i18n.getListFormatter({ style: "narrow" });
+    let selfItems = $derived(
+        self.items
+            .filter((i) => i.visible && i.matchScore > 0)
+            .sort((a, b) => b.matchScore - a.matchScore || a.name.localeCompare(b.name, game.i18n.lang)),
+    );
+    const withQuantity = (i: { quantity: number; marked: number; name: string }) =>
+        i.quantity > 1 ? `${i.marked}x ${i.name}` : i.name;
+    let itemsToSend = $derived.by(() => {
```

Comment:
Minor thing, feel free to resolve, but I think you can const the deriveds. You only need let if its a state or you plan to reassign it.